Snippets/Core/Core_6/Lifecycle/INeedInitialization.cs

```diff
@@ -12,9 +12,7 @@ public void Customize(EndpointConfiguration endpointConfiguration)
             // Perform initialization
             // This is after Type Scanning.
             // Do NOT call the following here:
-            // endpointConfiguration.ExcludeAssemblies();
```

Comment:
Not sure if we want to remove these just in case someone on < 6.2 is using INeedInitialization.